modules/admin-ui-frontend/.jshintrc

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
This file is required by the current state of `r/13.x`

Reply:
To be clear:
```
[INFO] 16 01 2023 12:33:13.247:INFO [framework.detect-browsers]: The following browsers will be used: [ 'ChromeHeadless', 'FirefoxHeadless' ]
[INFO] 16 01 2023 12:33:13.437:WARN [filelist]: Pattern "/home/greg/opencast/upstream/modules/admin-ui-frontend/test/test/unit/**/*Helper.js" does not match any file.
```

Reply:
Added it back, tests should run again now.